README.md

```diff
@@ -1,2 +1,61 @@
 # issue-sync
-A tool for synchronizing issues between GitHub and JIRA
+
+Issue-sync is an automated tool for synchronizing GitHub and JIRA issues.
```

Comment:
it;s not an automated tool until you add actual automation. It is automatable. just say `issue-sync is a tool for ...`